### ISSUE TITLE
Java error

### ISSUE THREAD

**Issue by chronoss09** [developersu/ns-usbloader#133]
![Capture d’écran 2023-02-13 124934](https://user-images.githubusercontent.com/17669980/218450627-547110b3-cbc0-44f2-8992-8e30be37153c.png)

Windows 11 and my Java is up date !!!

**Comment by developersu** [comment]
If you're running `jar`, go to cmd/powershell and run it via `java -jar ns-usbloader-7.0.jar`
Paste here output. There will be more information.

Or just use installer. It has java runtime inside.

**Comment by chronoss09** [comment]
![Capture](https://user-images.githubusercontent.com/17669980/218960615-acdea544-4e42-42be-a89f-c62d8eab611b.PNG)

**Comment by Masamune3210** [comment]
Update java

**Comment by developersu** [comment]
Yeah, just old JDK. 
To check you can type `java -version`. Install JDK 11 or later if you want to use `jar` instead of `exe`.

**Comment by dothackjhe** [comment]
> Yeah, just old JDK. To check you can type `java -version`. Install JDK 11 or later if you want to use `jar` instead of `exe`.

I followed this instruction, but I'm getting this error: https://github.com/developersu/ns-usbloader/issues/137

**Comment by dothackjhe** [comment]
> Update java, run the java uninstall tool first, it seems that somehow your path has a old version on it somewhere

I've only one instance of JRE installed after deleting the 32-bit one (JRE got duplicated). But it did not address the issue that OP raised, which I am also experiencing.

**Comment by DeadlyVoyage** [comment]
@developersu, why did you remove support for Java 8 Runtime? It doesn't make sense for me to install JDK just for this one app when the Java 8 Runtime supports MANY apps, including NS-USBloader v6.1 and lower. So why remove support and make things more complicated?

For anyone else having this issue, you have to either install JDK 11, or use the "Installer-7.0.exe" which will add more unnecessary bloat to your computer, or just keep using NS-USBloader v6.1 which works just fine with Java 8 Runtime. I'm just gonna keep using NS-USBloader v6.1. There is nothing new or notable in v7.0 to justify needing to install JDK 11. Just looks like the usually devs releasing an update just to release an update. SMH

**Comment by developersu** [comment]
@DeadlyVoyage because I don't know how to code using latest JDK LTE (e.g. Java 17) and make it work with JRE 8. Maybe I miss something and it's possible. If so, please let me know (!).
I prefer later versions of Java since this is what I need to learn better to find a job. Also it's kinda easier in some aspects like many nice features added since Java 8 (it has been released at 2014!), but yeah, that's not the main reason. 

Soooo the answer is somewhere between "I wanna use latest Java features" and "I don't know how to make it work with JRE 8". 

Surely your position make sense. Thank you for the feedback!

**Comment by DeadlyVoyage** [comment]
@developersu fair enough bro, at least you tried and have a good reason. I can respect that more than some other devs who only care about using the latest version just because they're the "latest version". Thanks for the explanation and for your hard work, it's much appreciated.
